packages/babel/src/transformation/file/index.js

```diff
@@ -301,7 +312,9 @@ export default class File {
   }
 
   /**
-   * [Please add a description.]
+   * Inserts an ImportDeclaration into the AST.
```

Comment:
Is the description I wrote accurate as far as it goes (a high level overview)? I wasn't trying to comprehensively document these things, just get the ball rolling. It takes a lot of work to figure out what the methods do, especially starting from scratch with no comments. I think an incremental approach to adding this documentation will be more viable. But let me know if this requires more detail for right now and I'll dig into it more.